src/IO/Exporter/Exporter.hpp

```diff
@@ -49,11 +49,11 @@ struct ObservationStep {
  * of the domain (default is `false`). This can be useful to fill the excision
  * region with (constraint-violating but smooth) data so it can be imported into
  * moving puncture codes. Specifically, we implement the strategy used in
- * \cite Etienne2008re adjusted for distorted excisions: we choose uniformly
+ * \cite Etienne2008re adjusted for distorted excisions: we choose 8 uniformly
  * spaced radial anchor points spaced as $\Delta r = 0.3 r_\mathrm{AH}$ in the
  * grid frame (where the excision is spherical), then map the anchor points to
- * the distorted frame (where we have the target point) and do a 7th order
- * polynomial extrapolation into the excision region.
+ * the distorted frame (where we have the target point) and do a polynomial
+ * extrapolation into the excision region.
```

Comment:
Is it 8th order or not?

Reply:
It's a 7th order polynomial with 8 collocation points. However in the `polynomial_interpolation` function the name `order` is `degree + 1`:
https://github.com/sxs-collaboration/spectre/blob/bc197745bb08b2397fb31dda402a10b4d17cc856/src/NumericalAlgorithms/Interpolation/PolynomialInterpolation.cpp#L19
To avoid confusion I renamed to `num_extrapolation_anchors`.